manifests/namespace-install.yaml

```diff
@@ -246,17 +246,16 @@ data:
           metricsExporterImage: natsio/prometheus-nats-exporter:latest
           configReloaderImage: natsio/nats-server-config-reloader:latest
           startCommand: nats-server
-        - version: 2.7.4
-          natsImage: nats:2.7.4
+        - version: 2.8.1
+          natsImage: nats:2.8.1
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
           configReloaderImage: natsio/nats-server-config-reloader:0.6.3
           startCommand: /nats-server
-        - version: 2.7.4-alpine
-          natsImage: nats:2.7.4-alpine
+        - version: 2.8.1-alpine
+          natsImage: nats:2.8.1-alpine
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
```

Comment:
In the future, I imagine we'll add versions rather than replacing the existing versions, right?

Reply:
yes.